src/test/scala/cognite/spark/v1/DataPointsRelationTest.scala

```diff
@@ -966,6 +937,81 @@ class DataPointsRelationTest extends FlatSpec with Matchers with ParallelTestExe
     assert(error.getMessage.contains(s"The timeseries id=$valhallStringTimeSeriesId"))
   }
 
+  it should "read and write datapoints in the future" taggedAs WriteTest in {
+    val testUnit = s"future ${shortRandomString()}"
+    val tsName = s"future${shortRandomString()}"
```

Comment:
should we delete the timeseries at the end of the test, to avoid having a lot of random timeseries in the tenant?

Reply:
right, done here https://github.com/cognitedata/cdp-spark-datasource/pull/535/commits/982f286fbeb7e39678f5e8b31f575b6da4a1a5cf